Automattic/wp-calypso
# Site Profiler: fix email check on site profiler

Related to #82127

## Proposed Changes

* Follow up from previous PR, in some cases the whois field will be empty and we need to make sure the value exists before doing any further checks.

## Testing Instructions



* Go to `http://calypso.localhost:3000/site-profiler`.
* Fill in something that can't be identified by WHOIS parser, something like `aabbcct20230926.tw`
* Make sure the screen still shows fine and not a blank page.

## Pre-merge Checklist
